test/unit/cfg_errors_test.lua

```diff
@@ -238,18 +238,26 @@ g.test_console_sock = function()
     g.mock_membership()
 
     local sock_dir = fio.pathjoin(g.tempdir, 'sock')
+    local sock_name = string.format('%s/%s.sock', sock_dir, ('a'):rep(110))
     fio.mktree(sock_dir)
 
-    check_error('Too long console_sock exceeds UNIX_PATH_MAX limit',
-        cartridge.cfg, {
-            workdir = '/tmp',
-            advertise_uri = 'unused:0',
-            http_enabled = false,
-            roles = {},
-            console_sock = string.format('%s/%s.sock', sock_dir, ('a'):rep(110)),
-        }
-    )
+    local ok, err = cartridge.cfg({
+        workdir = '/tmp',
+        advertise_uri = 'unused:0',
+        http_enabled = false,
+        roles = {},
+        console_sock = sock_name,
+    })
+
+    t.assert_not(ok)
+    t.assert_covers(err, {class_name = 'CartridgeCfgError'})
     t.assert_equals(fio.listdir(sock_dir), {})
+    local e1 = 'Too long console_sock exceeds UNIX_PATH_MAX limit'
```

Comment:
I suggest you to add some comments that one line is needed for "before Tarantool 2.3" and another for "Tarantool 2.3+"

Reply:
Done